front/lib/actions/mcp_actions.ts

```diff
@@ -133,31 +140,6 @@ function makeMCPConfigurations({
   });
 }
 
-export const getServerTypeAndIdFromSId = (
```

Comment:
moved to another file to avoid cyclic dependencies (it's needed in MCPServerConnectionResource)